modules/services/elasticsearch/main.tf

```diff
@@ -14,24 +14,21 @@ resource aws_elasticsearch_domain "openedx" {
   elasticsearch_version = "1.5"
 
   cluster_config {
-    instance_count = 2
+    instance_count = var.instance_count
     instance_type = var.elasticsearch_instance_type
-    zone_awareness_enabled = true
+    zone_awareness_enabled = var.zone_awareness_enabled
 
-    dedicated_master_enabled = true
+    dedicated_master_enabled = var.dedicated_master_enabled
     dedicated_master_type = var.elasticsearch_instance_type
     dedicated_master_count = var.number_of_nodes
 
     zone_awareness_config {
-      availability_zone_count = 2
+      availability_zone_count = var.availability_zone_count
     }
   }
 
   vpc_options {
-    subnet_ids = [
-      tolist(data.aws_subnet_ids.default.ids)[1],
-      tolist(data.aws_subnet_ids.default.ids)[2],
-    ]
+    subnet_ids = slice(tolist(data.aws_subnet_ids.default.ids), var.start_number_of_subnets, var.number_of_subnets)
```

Comment:
Are we ever going to use a subset of the `data.aws_subnet_ids` here? If so, then we're unlikely to be able to guarantee that the subnets we want are sequential in this list.

So I'd say either:
* Use all the subnets in `data.aws_subnet_ids`, OR
* Make the `subnet_ids` an input variable.

Reply:
@pomegranited you are right :+1: I set all subnets by default and now you can specify the subnet IDs as well